### ISSUE TITLE
Is it possible to enable hyperlinks in the rendered PDF?

### ISSUE THREAD

**Issue by nessgor** [VadimDez/ng2-pdf-viewer#18]
https://github.com/mozilla/pdf.js/blob/master/examples/components/simpleviewer.js#L37

```
// (Optionally) enable hyperlinks within PDF files.
var pdfLinkService = new PDFJS.PDFLinkService();
```

Is it possible to show the hyperlinks in proper annotated PDF ? 

other related post:
http://stackoverflow.com/questions/37016124/display-annotations-hyperlinks-using-pdf-js

http://stackoverflow.com/questions/33813373/pdf-js-how-to-open-hyperlinks-in-a-new-tab-window

**Comment by nessgor** [comment]
[test.pdf](https://github.com/VadimDez/ng2-pdf-viewer/files/485879/test.pdf)

Using the browser below can check whether the hyperlink of the PDF is properly annotated
https://brendandahl.github.io/pdf.js.utils/browser/

![1](https://cloud.githubusercontent.com/assets/11316684/18726174/d3942512-8075-11e6-860b-d244b83e56ac.PNG)

**Comment by VadimDez** [comment]
fixed by 2.0.0
